app.logorrr.LogoRRR.yml

```diff
@@ -0,0 +1,36 @@
+app-id: app.logorrr.LogoRRR
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: LogoRRR
+modules:
+  - name: LogoRRR
+    only-arches: [aarch64]
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/LogoRRR/
+      - mkdir -p /app/bin
+      - cp -r binaries/LogoRRR/* /app/lib/LogoRRR/
+      - ln -s /app/lib/LogoRRR/bin/LogoRRR /app/bin/LogoRRR
+      - install -Dm644 icons/logorrr-icon-16.png /app/share/icons/hicolor/16x16/apps/app.logorrr.LogoRRR.png
+      - install -Dm644 icons/logorrr-icon-32.png /app/share/icons/hicolor/32x32/apps/app.logorrr.LogoRRR.png
+      - install -Dm644 icons/logorrr-icon-64.png /app/share/icons/hicolor/64x64/apps/app.logorrr.LogoRRR.png
+      - install -Dm644 icons/logorrr-icon-128.png /app/share/icons/hicolor/128x128/apps/app.logorrr.LogoRRR.png
+      - install -Dm644 icons/logorrr-icon-256.png /app/share/icons/hicolor/256x256/apps/app.logorrr.LogoRRR.png
+      - install -Dm644 icons/logorrr-icon-512.png /app/share/icons/hicolor/512x512/apps/app.logorrr.LogoRRR.png
+      - install -Dm644 app.logorrr.LogoRRR.metainfo.xml /app/share/metainfo/app.logorrr.LogoRRR.metainfo.xml
+      - install -Dm644 app.logorrr.LogoRRR.desktop -t /app/share/applications
+    sources:
+      - type: file
+        path: app.logorrr.LogoRRR.desktop
+      - type: file
+        path: app.logorrr.LogoRRR.metainfo.xml
+      - type: archive
+        url: https://www.logorrr.app/downloads/25.1.0/flatpak-archive-25.1.0.tar.gz
+        sha256: 47d5e0b8b7bfc01bed6c06b7d66e775a333dfa3cfdf58c663f7cf427c5bede13
+
+finish-args:
```

Comment:
`finish-args` is always above `modules`

Reply:
i put it above the modules section. maybe things like these could be catched by the linter?